### DISCUSSION TITLE
Getting problem with GRIDobj

### DISCUSSION THREAD

**Discussion by premhrc** [wschwanghart/topotoolbox#51] [Q&A] [answered]
After adding path of topotoolbox 
getting this error
>> dem_fname="Baspa_SRTM1_30m_UTM46N.tif";
>> Baspa_dem = GRIDobj(dem_fname);
Error using exist
The first input to exist must be a string scalar or character vector.

Error in GRIDobj (line 277)
                        tfwfile_exists = exist(tfwfile,'file');

**Comment by wschwanghart** [comment, marked as answer]
Oh, yes, this is an error that recently showed up. I am working on it. In the meantime, following code should work:
```
Baspa_dem = GRIDobj(char(dem_fname));
```
Best regards.
